server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/UnsetDefaultAction.java

```diff
@@ -49,9 +53,6 @@ public void define(WebService.NewController controller) {
   }
 
   @Override
-  public void handle(Request request, Response response) {
-    qualityGates.setDefault(null);
-    response.noContent();
+  public void handle(Request request, Response response) throws Exception {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1186)